model-files/RunModel.bat

```diff
@@ -369,6 +369,10 @@ if ERRORLEVEL 2 goto done
 :here
 IF %ITERATION% LSS %MAX_ITERATION% GOTO iteration_start
 
+:: Create the block file that controls whether the crowding functions are called during transit assignment.
+"%PYTHON_PATH%"\python.exe %BASE_SCRIPTS%\assign\transit_assign_type_set.py CTRAMP\scripts\block\transit_assign_type.block
+
```

Comment:
Where is transit_assign_type_set.py ?
Also, please check for errors after running the script

Reply:
I missed adding the file to git in my previous check-in.